2025-05-27 11:35:22.881079+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "DEPARTMENT" が見つかりません
Column "DEPARTMENT" not found; SQL statement:
ALTER TABLE users DROP COLUMN department [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:775)
	at org.h2.command.Parser.parseAlterTableDrop(Parser.java:8378)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8155)
	at org.h2.command.Parser.parseAlter(Parser.java:7211)
	at org.h2.command.Parser.parsePrepared(Parser.java:650)
	at org.h2.command.Parser.parse(Parser.java:592)
	at org.h2.command.Parser.parse(Parser.java:564)
	at org.h2.command.Parser.prepareCommand(Parser.java:483)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:639)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:559)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1166)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:245)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1345)
	at org.h2.server.web.WebApp.query(WebApp.java:1143)
	at org.h2.server.web.WebApp.query(WebApp.java:1119)
	at org.h2.server.web.WebApp.process(WebApp.java:245)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:177)
	at org.h2.server.web.JakartaWebServlet.doGet(JakartaWebServlet.java:129)
	at org.h2.server.web.JakartaWebServlet.doPost(JakartaWebServlet.java:166)
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:108)
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:231)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:365)
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:100)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:131)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:85)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:179)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at com.example.backend.security.JwtAuthenticationFilter.doFilterInternal(JwtAuthenticationFilter.java:61)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:107)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:93)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.web.filter.CorsFilter.doFilterInternal(CorsFilter.java:91)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:82)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:62)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:233)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:191)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.servlet.handler.HandlerMappingIntrospector.lambda$createCacheFilter$3(HandlerMappingIntrospector.java:195)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:74)
	at org.springframework.security.config.annotation.web.configuration.WebMvcSecurityConfiguration$CompositeFilterChainProxy.doFilter(WebMvcSecurityConfiguration.java:230)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:352)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:268)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:482)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:389)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:896)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-05-27 16:12:09.322531+09:00 jdbc[85]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-27 16:12:09.606923+09:00 jdbc[85]: exception
org.h2.jdbc.JdbcSQLDataException: 列 "TRANSACTION_TYPE CHARACTER VARYING(10)" の値が長過ぎます: "'MANUAL_RECEIVE' (14)"
Value too long for column "TRANSACTION_TYPE CHARACTER VARYING(10)": "'MANUAL_RECEIVE' (14)"; SQL statement:
insert into inventory_transaction (manufacturer,operator,order_no,purchase_price,quantity,remarks,item_code,supplier,transaction_time,transaction_type,transaction_id) values (?,?,?,?,?,?,?,?,?,?,default) [22001-224]
2025-05-27 16:15:23.673572+09:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-27 16:15:23.680625+09:00 jdbc[95]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-27 16:21:21.726570+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-27 16:21:21.789584+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 05:50:51.198956+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 05:53:43.955150+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 05:57:38.010230+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 05:57:59.076766+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 06:01:36.853383+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 06:01:36.885299+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 06:08:53.609310+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 06:08:53.611311+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 07:58:51.587864+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 07:58:51.623616+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:16:07.436182+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:16:07.436182+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:17:29.841977+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:17:29.846434+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:26:43.629750+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:26:43.648902+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:41:14.763714+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:41:14.779534+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:42:26.868620+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:42:26.873688+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:44:25.437124+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 08:44:25.437124+09:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:02:02.825255+09:00 jdbc[23]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:02:02.841596+09:00 jdbc[23]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:28:51.433268+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:28:51.450630+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:32:00.129029+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:32:00.133857+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:40:27.447664+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:40:27.463580+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:41:00.974325+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:41:00.975852+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:42:08.961958+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:42:08.978492+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:42:49.974517+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:42:49.990128+09:00 jdbc[33]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:49:06.693800+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 09:49:48.136701+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:04:08.377144+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:04:08.393819+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:06:46.848980+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:06:46.850990+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:09:30.714676+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:09:30.716977+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:17:13.079387+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:17:13.091488+09:00 jdbc[43]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:24:17.794424+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:24:17.820366+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:26:40.657125+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:26:40.657125+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:27:14.020852+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:27:14.020852+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:28:19.848500+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:28:19.848500+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:42:00.430799+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:42:00.435011+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:43:45.682803+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 10:43:45.736321+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-2729763C' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 17) NOT NULL)"
Data conversion error converting "'PO20250528-2729763C' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 17) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 11:30:14.456273+09:00 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 11:30:14.614506+09:00 jdbc[24]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-30A6EF23' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 18) NOT NULL)"
Data conversion error converting "'PO20250528-30A6EF23' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 18) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 11:31:21.527001+09:00 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 11:31:21.533014+09:00 jdbc[24]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-391919E7' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 19) NOT NULL)"
Data conversion error converting "'PO20250528-391919E7' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 19) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 11:32:02.098616+09:00 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 11:32:20.459315+09:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 11:32:20.463319+09:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-949116BA' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 21) NOT NULL)"
Data conversion error converting "'PO20250528-949116BA' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 21) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 11:33:59.700983+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 11:33:59.792641+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-82F8D08F' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 22) NOT NULL)"
Data conversion error converting "'PO20250528-82F8D08F' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 22) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 11:57:32.456006+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 12:00:51.655582+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 12:00:51.740133+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-93799070' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 24) NOT NULL)"
Data conversion error converting "'PO20250528-93799070' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 24) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 12:02:25.414248+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 12:02:25.517580+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'PO20250528-2F1D6BC2' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 25) NOT NULL)"
Data conversion error converting "'PO20250528-2F1D6BC2' (PURCHASE_ORDER: ""ID"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1 RESTART WITH 25) NOT NULL)"; SQL statement:
insert into purchase_order_detail (category,item_code,item_name,model_number,order_no,purchase_price,quantity,received_quantity,remarks,status,id) values (?,?,?,?,?,?,?,?,?,?,default) [22018-224]
2025-05-28 13:23:58.480180+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:25:34.454683+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:25:34.538162+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: 列 "TRANSACTION_TYPE CHARACTER VARYING(10)" の値が長過ぎます: "'MANUAL_RECEIVE' (14)"
Value too long for column "TRANSACTION_TYPE CHARACTER VARYING(10)": "'MANUAL_RECEIVE' (14)"; SQL statement:
insert into inventory_transaction (manufacturer,operator,order_no,purchase_price,quantity,remarks,item_code,supplier,transaction_time,transaction_type,transaction_id) values (?,?,?,?,?,?,?,?,?,?,default) [22001-224]
2025-05-28 13:26:41.084551+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:26:41.084551+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:28:23.773392+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:28:23.785392+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:28:23.789392+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:28:57.794566+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:35:54.421101+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:35:54.426756+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:36:05.599333+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:37:01.245146+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:37:01.245146+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:38:45.245440+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:38:45.251446+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
2025-05-28 13:38:45.253860+09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: オブジェクトはすでに閉じられています
The object is already closed [90007-224]
